.goreleaser.docker.yml

```diff
@@ -45,7 +45,7 @@ dockers:
       - &amd_debug_image_gh "ghcr.io/authzed/zed:v{{ .Version }}-amd64-debug"
       - &amd_debug_image_dh "authzed/zed:v{{ .Version }}-amd64-debug"
     ids: ["linux-amd64"]
-    dockerfile: &dockerfile "Dockerfile.release"
```

Comment:
This was a copy-paste that was redundant - we've already got this def further up.